concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md

```diff
@@ -0,0 +1,128 @@
+Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code in.
+
+The following code displays a notification every time users change the ColorBox value. The notification's background assumes the picked color. 
```

Comment:
`The notification's background assumes the picked color.`

Can we say the following instead?

`The selected color is applied to the notification's background.`

Reply:
Yes, I changed it

Reply:
Thank you!